app/views/user_mailer/send_flag_notification.html.erb.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
The file should be named `app/views/user_mailer/send_flag_notification.html.erb` not `app/views/user_mailer/send_flag_notification.html.erb.erb` (extra .erb at the end)